sdk/go/auto/cmd_windows.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build windows
+// +build windows
+
+package auto
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/iwdgo/sigintwindows"
+)
+
+func interruptProcess(proc *os.Process) error {
+	return sigintwindows.SendCtrlBreak(proc.Pid)
+}
+
+func setSysprocAttrNewProcessGroup(cmd *exec.Cmd) {
+	cmd.SysProcAttr = &syscall.SysProcAttr{
+		CreationFlags: syscall.CREATE_NEW_PROCESS_GROUP,
```

Comment:
We apparently need to create a new process group here explicitly, otherwise at least in the test, the test also gets killed with an interrupt.  This isn't an issue on MacOS or Linux in the tests, so we assign nothing there.